Docs/Sample.plist

```diff
@@ -1156,32 +1158,8 @@
 		</dict>
 		<key>Serial</key>
```

Comment:
I removed the `Override` option here too. However, OcSerializeLib will complain now. Is it possible to make BOOLEAN optional as well?